apps/mobile/src/libs/packages/store/store.package.ts

```diff
@@ -37,13 +41,15 @@ class Store {
 			devTools: config.ENV.APP.ENVIRONMENT !== AppEnvironment.PRODUCTION,
 			middleware: (getDefaultMiddleware) => {
 				return getDefaultMiddleware({
+					serializableCheck: false,
```

Comment:
What does this do?

Reply:
This change turns off the `serializableCheck` in Redux. Normally, it ensures our actions and state are simple, which helps keep Redux predictable. However, without turning it off, the emulator was having issues with more complex data like dates or special objects. By disabling `serializableCheck`, we avoid these problems and keep the emulator running smoothly.

Reply:
What were the issues with the emulator? We shouldn't turn off serializableCheck

Reply:
I agree! We are not building the app for emulators. If this doesn't cause issues on real devices we shouldn't turn it off.